### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my component- Avatar!

### CODE REVIEW

File: Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  padding: 0;
+}
+
+:root{
+    --gray:#3e4152;
+    --text-color:#007FFF;
+    --blue-bg:#023e8a;
+    --bg-code:#FAFAFA;
+    --colour-txt:white;
+}
```
Comment (Naveenchandar) [inline Avatar/avatar.css:14]: Suggestion: 
Please follow hex code for consistency and rename your variables for better readability.

File: Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+h2{
+    font-size: 2rem;
+}
+p{
+    font-size: 1.2rem;
+    padding: 0.8rem;
+}
+.img-box, .text-box, .square-box{
```
Comment (Naveenchandar) [inline Avatar/avatar.css]: You can use common class name to elements instead of multiple classes.

**Comment by Rohitprasad83** [comment]
Looks good!

**Review by Naveenchandar — APPROVED** [review]
Looks good.

**Review by Rohitprasad83 — APPROVED** [review]

File: Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    padding: 1.5rem;
+    border-radius: 6px;
+    margin-bottom: 1.5rem;
+    display: flex;
+    flex-wrap: wrap;
+}
+.fas{
+    font-size: 1rem;
+    padding: 0.4rem;
+}
```
Comment (rohanmathur91) [inline Avatar/avatar.css:50]: Cant understand what is fas ?

File: Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    width: 6.9rem;
+    height:6.9rem;
+    font-size: 1.5rem;
+}
+.avatar-xs-size{
+    margin-top: 4.5rem;
+    width: 4.9rem;
+    height:4.9rem;
+    font-size: 0.8rem;
+}
```
Comment (rohanmathur91) [inline Avatar/avatar.css:83]: No need to specifically mention size after the class name ie avatar-sm-size, avatar-sm also works well as it is a standard naming convention for sizes.

**Review by rohanmathur91 — COMMENTED** [review]
Looks good.

**Pull request merged**